Gemfile

```diff
@@ -6,4 +6,6 @@ git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 gemspec
 
 gem 'httparty', '~> 0.15.6'
+gem 'pry'
```

Comment:
Might move this to a `development` section so we don't have pry floating around.